python/src/file_io.py

```diff
@@ -14,19 +14,33 @@ def write(self, content: str) -> None:
     def read(self) -> str:
         raise NotImplementedError
 
+    @staticmethod
+    def from_json(json_dict):
+        if "__File__" in json_dict:
+            return File.from_json(json_dict)
+        raise TypeError
```

Comment:
Don't think we need this here? Just on `File`?

Reply:
Yeah, since when parsing it doesn't know which child it is, the BaseNetlist thing just has an IFile that could be a mock file